lib/broadside/configuration/deploy.rb

```diff
@@ -22,14 +22,18 @@ class DeployConfig < ConfigStruct
         :env_vars,
```

Comment:
heh, that's a lot of work - i wouldn't worry about speccing `ecs.rb` for now, let's talk about that separately